aep/general/0126/aep.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 id: 126
 state: approved
+slug: enums
```

Comment:
I would rather type "enums"  in a short link, but we can add alias short links for that...for a primary identifier that shows up outside short links, is the unabbreviated "enumerations" more appropriate?

Reply:
Sounds good! agreed.